### ISSUE TITLE
💅 `useExpect` false positives on Chai-style `assert` calls

### ISSUE THREAD

**Issue by Bertie690** [biomejs/biome#9172]
### Environment information

<details>

```bash

N/A
```

</details>

### Rule name

`useExpect`

### Playground link

https://biomejs.dev/playground/?lineWidth=120&indentStyle=space&lintRules=useExpect&analyzerFixMode=safeAndUnsafeFixes&code=aQB0ACgAIgArACIALAAgACgAKQAgAD0APgAgAHsACgAgACAAYQBzAHMAZQByAHQAKAAxACAAKwAgADIAIAA9AD0APQAgADIAIAArACAAMQAsACAAIgBhAGQAZABpAG4AZwAgAHMAaABvAHUAbABkACAAYgBlACAAYwBvAG0AbQB1AHQAYQB0AGkAdgBlACIAKQAKACAAIABhAHMAcwBlAHIAdAAoADEAIAArACAAKAAyACAAKwAgADMAKQAgAD0APQA9ACAAKAAxACAAKwAgADIAKQAgACsAIAAzACwAIAAiAGEAZABkAGkAbgBnACAAcwBoAG8AdQBsAGQAIABiAGUAIABhAHMAcwBvAGMAaQBhAHQAaQB2AGUAIgApAAoAIAAgAGEAcwBzAGUAcgB0ACgAMQAgACsAIAAoADIAIAAqACAAMwApACAAPQA9AD0AIAAoADEAIAAqACAAMgApACAAKwAgACgAMQAgACoAIAAzACkALAAgACIAYQBkAGQAaQBuAGcAIABzAGgAbwB1AGwAZAAgAGQAaQBzAHQAcgBpAGIAdQB0AGUAIABvAHYAZQByACAAbQB1AGwAdABpAHAAbABpAGMAYQB0AGkAbwBuACIAKQAKAH0AKQA%3D

### Expected result

It should not error, as Vitest re-exports an alternative testing syntax (`assert`) for chai-style asserts.
These are still assertions that test something about the code, and should be counted.

Related to #9168 (and can likely be fixed in 1 fell swoop)

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by Bertie690** [comment]
I am claiming this and fixing it at the same time as #9168 btw
